source/form.html

```diff
@@ -0,0 +1,173 @@
+      </div>
+    </nav>
+  </header>
+  <main class="main-container">
+    <section class="welcome-block">
+      <h1 class="visually-hidden">Седона</h1>
+      <img class="sedona-image" src="#" width="0" height="0" alt="#">
+    </section>
+    <section class="feedback">
+      <header class="feedback-header">
+        <h2 class="feedback-header__title">Оставьте свой отзыв</h2>
+        <p class="feedback-header__text">Помогите нашим отелям стать лучше! оставьте отзыв о них, а также о посещенных вами
+          достопримечательностях</p>
+      </header>
+      <form class="feedback-form" action="https://echo.htmlacademy.ru/" method="post">
+        <fieldset class="feedback-form__group">
+          <legend class="feedback-form__title">Представьтесь:</legend>
+          <p class="feedback-form__wrapper-name">
+            <label class="feedback-form__label" for="visitors-name">ИМЯ*:</label>
+            <input class="feedback-form__input" type="text" id="visitors-name" name="visitors-name" placeholder="Пётр">
+          </p>
+          <p class="feedback-form__surname">
+            <label class="feedback-form__label" for="visitors-surname">ФАМИЛИЯ*:</label>
+            <input class="feedback-form__input" type="text" id="visitors-surname" name="visitors-surname" placeholder="Пустота">
+          </p>
+          <p class="feedback-form__patronymic">
+            <label class="feedback-form__label" for="visitors-patronymic">ОТЧЕСТВО:</label>
+            <input class="feedback-form__input" type="text" id="visitors-patronymic" name="visitors-patronymic" placeholder="Викторович">
+          </p>
+        </fieldset>
+
+        <fieldset class="feedback-form__group">
+          <legend class="feedback-form__title">Контактная информация:</legend>
+          <p class="feedback-form__phone">
+            <label class="feedback-form__label" for="contact-phone">Контактный телефон*:</label>
+            <input class="feedback-form__input" type="tel" id="contact-phone" name="contact-phone" placeholder="Введите телефон">
+          </p>
+          <p class="feedback-form__email">
+            <label class="feedback-form__label" for="email">Электронная почта*:</label>
+            <input class="feedback-form__input" type="email" id="email" name="email" placeholder="Введите e-mail">
+          </p>
+        </fieldset>
+
+
+
+        <fieldset class="feedback-form__group">
+          <legend class="feedback-form__title">Ваше общее впечатление:</legend>
+          <ul class="feedback-form__list">
+            <li class="feedback-form__item">
+              <label class="control">
+                <input class="control-input" type="radio" name="Скорее положительное" checked>Скорее положительное
+              </label>
+            </li>
+            <li class="feedback-form__item">
+              <label class="control">
+                <input class="control-input" type="radio" name="Скорее отрицательное">Скорее отрицательное
+              </label>
```

Comment:
Когда есть и элемент и текст, то тоже стоит делать их на разных строках.
Потому что текст где-то в конце строки после закрытия тега сложно замечать

```suggestion
              <label class="control">
                <input class="control-input" type="radio" name="Скорее отрицательное">
                Скорее отрицательное
              </label>
```